ports/clashmastery/port.json

```
@@ -0,0 +1,28 @@
+{
+    "version": 3,
+    "name": "clashmastery.zip",
+    "items": [
+        "Clash Mastery.sh",
+        "clashmastery"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Clash Mastery",
+        "porter": [
+            "tabreturn"
+        ],
+        "desc": "This boss fight is too hard. You'll need some practice. Master each attack pattern, one-by-one. Practice makes you tough!",
+        "desc_md": null,
+        "inst": "Ready to run. Thanks to Abhi Sundu (https://abhimonk.itch.io/) for creating this great game and kindly granting PortMaster permission to distribute the game files.",
+        "inst_md": "Ready to run. Thanks to [Abhi Sundu](https://abhimonk.itch.io/) for creating this great game and kindly granting PortMaster permission to distribute the game files.",
+        "genres": [
+            "action"
+        ],
+        "image": {},
+        "rtr": true,
+        "exp": false,
+        "runtime": "love2d",
```

love2d is included with PM-GUI [see here](https://github.com/PortsMaster/PortMaster-GUI/tree/main/PortMaster/runtimes/love_11.5)
```suggestion
        "runtime": null,
```